deploy/skaffold/Dockerfile

```diff
@@ -12,19 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# First stage is to build the skaffold binary from source
-FROM golang:1.10 as build
-WORKDIR /go/src/github.com/GoogleCloudPlatform/skaffold
-COPY . .
-RUN make
-
 # Second stage is used to install the binaries that we need
 FROM gcr.io/gcp-runtimes/ubuntu_16_0_4
+
 RUN apt-get update && \
     apt-get install --no-install-recommends --no-install-suggests -y \
-        curl \
         ca-certificates \
-        git
+        curl \
+        build-essential \
+        git \ 
+        gcc \
+        python-dev \
+        python-setuptools \
+        apt-transport-https \
+        lsb-release
+
+COPY --from=golang:1.10 /usr/local/go /usr/local/go
```

Comment:
Go isn't in the ubuntu package manager anywhere?

Reply:
We can't ensure that we use the same version that way

Reply:
Ah, makes sense.